### ISSUE TITLE
Font names with spaces require a comma

### ISSUE THREAD

**Issue by Ry7n** [andymeneely/squib#138]
I was struggling a bit to get font names with spaces to work. Some of your examples:

```ruby
text str: "Hello", font: 'Times New Roman,Sans 25'
```

Wouldn't work on OS X.

The solution I found was *not* to specify alternates, but to terminate the font family name with a comma, e.g. 

```ruby
text str: "Hello", font: 'Times New Roman, 25'
text str: "Hello", font: 'Times New Roman, Bold 25'
```

I discovered this based on the [this](http://www.graphviz.org/mantisbt/view.php?id=1823) similar graphviz issue.

**Comment by andymeneely** [comment]
Yeah, I'm aware of this and I don't know that I can do anything about it. I've noticed different behaviors on different OS's, so it might even go deeper than Pango.

I've also upgraded Pango using `brew` and had this behavior change, too. OSX's built-in Pango is kinda old.

I think our only option is to just document the behavior. I'll document this behavior in our (in progress) docs rewrite. If you would like to contribute, just write some words on this bug and I'll put into the docs.
